### PR TITLE
Login branch

### PR DESCRIPTION
**Pull request by Jisan129**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 
 app.get("/express",((req, res) => {
     console.log("Sending data");
+    res.send('hello world');
+/*
+    This sends data to react
     res.json({helloWorld: "hello world"});
+*/
 }))
+app.use('/',loginRouter);
```
Comment (Labnann) [inline backend/app.js:16]: We use routes like: /authenticate/login/
or something like this.

File: backend/route/login.controller.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const route = express.Router()
+route.get('/', (req, res) => {
```
Comment (Labnann) [inline backend/route/login.controller.js]: same goes for here.

File: backend/route/login.route.js [hunk 3]
```diff
@@ -4,14 +4,14 @@ const app = express();
...
 route.get('/', (req, res) => {
     console.log("Done my boi")
     res.send("done")
     res.end()
 })
-route.post('/',(req,res) => {
-    const name = req.body.name
-    res.send(name)
-})
+route.post('/',checkLoginData,confirmLogin)
```
Comment (Labnann) [inline backend/route/login.route.js:15]: I like how you are handling this multiple middlewares O.O It is really cool.

**Review by Labnann — CHANGES_REQUESTED** [review]
Everything looks great. I think  I should merge this after the routes are set to /authenticate/login or something.

File: backend/route/login.route.js (hunk 3 above)
Comment (Jisan129) [reply backend/route/login.route.js]: Check it now

File: backend/route/api/authenticate/login.route.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const route = express.Router()
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({extended:false}))
+
+const checkLoginData=require('../../../middleware/login.middleware')
+const confirmLogin=require('../../../controller/login.controller')
+
+route.get('/', (req, res) => {
```
Comment (Labnann) [inline backend/route/api/authenticate/login.route.js:10]: Here too

File: backend/route/api/authenticate/login.route.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    console.log("Done my boi")
+    res.send("done")
+    res.end()
+})
+route.post('/',checkLoginData,confirmLogin)
```
Comment (Labnann) [inline backend/route/api/authenticate/login.route.js:15]: this too

**Review by Labnann — CHANGES_REQUESTED** [review]

**Review by Labnann — APPROVED** [review]
Good. Merging.

**Pull request merged**